dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -870,6 +871,26 @@
         }
       }
     },
+    "RegexKey": {
+      "description": "The regex key extra data kind.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "RegexKey",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "The key of the regex pattern used in extracting the entity."
+        },
+        "regexPattern": {
+          "type": "string",
+          "description": "The regex pattern used in extracting the entity."
```

Comment:
You should describe what sort of regex you support e.g., .NET (with a link), POSIX, etc. So many different "standards".

Reply:
I definitely can describe what sort of regex we support. But wouldn't it be more appropriate for that to be done on the authoring side?
The pattern we're returning here is just relaying the information that the span was detected by this pattern, and not for the purposes of being executed by the customer if that makes sense.

Let me know if you still would like me to add descriptions about what kind of regex we support. It's .NET.

Reply:
If the pattern isn't necessary to be parsed by people/processes, then why return it? Seems to me, if you're returning it, it must be useful information. So describing what grammar of regex it is would be useful too, right? Afterall, bash/dotnet/perl/vim/etc are all very different grammars.

It's just a doc string, so I don't see harm in including both here and on the authoring side.

Reply:
Sounds good. Doc string modified.